src/modules/presence/presence.c

```diff
@@ -1854,6 +1854,20 @@ static const char *rpc_presence_cleanup_doc[3] = {
 		"presentity, and watchers tables.",
 		0};
 
+
+void rpc_presence_htable_db_restore(rpc_t *rpc, void *c)
+{
+	LM_DBG("Restoring presence publ cache from database.\n");
+	pres_htable_db_restore();
```

Comment:
This functions returns -1 on errors and you need to catch that and return a proper return code on success and errors.